### PR TITLE
Different tweets

### PR DESCRIPTION
**Pull request by MehulKChaudhari**
This PR adds a feature, users will get different tweets on clicking  `tweet this out` button

### CODE REVIEW

**Review by sushilburagute — APPROVED** [review]
Everything checks out! It looks like a promising feature 💯

**Comment by MehulKChaudhari** [comment]
> Everything checks out! It looks like a promising feature

Thanks for the review Bro

**Comment by uditkumar01** [comment]
LGTM :fire:

**Comment by akankshach29** [comment]
Please change "resubmission" to "resubmitted" in {status !== 'portfolio_needs_revision' ? 'submitted' : 'resubmission'}!

File: client/src/components/CongratsCard/CongratsCard.tsx [hunk 1]
```diff
@@ -22,7 +25,7 @@ export function CongratsCard({
     >
       <Heading as="h1" size="lg" color="#00F0FF" fontFamily="Inter">
         Yay! Portfolio is successfully{' '}
-        {status !== 'needs revision' ? 'submitted' : 'resubmission'}! 🎉
+        {status !== 'portfolio_needs_revision' ? 'submitted' : 'resubmission'}! 🎉
```
Comment (akankshach29) [inline client/src/components/CongratsCard/CongratsCard.tsx:28]: replace "resubmission" with "resubmitted"

Comment (MehulKChaudhari) [reply client/src/components/CongratsCard/CongratsCard.tsx]: Done. Thanks for the review 🙂

**Review by akankshach29 — APPROVED** [review]

**Pull request merged**